recipes/root/all/conanfile.py

```diff
@@ -0,0 +1,316 @@
+import os
+import shutil
+import stat
+from glob import glob
+
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class PythonOption:
+    OFF = "off"
+    SYSTEM = "system"
+    # in future we may allow the user to specify a version when
+    # libPython is available in Conan Center Index.
+    # FIXME: add option to use CCI Python package when it is available
+    ALL = [OFF, SYSTEM]
+
+
+required_conan_version = ">=1.29.1"
+
+
+class RootConan(ConanFile):
+    name = "root"
```

Comment:
This is a very generic name 🤔

Reply:
Where a ROOT recipe has been added to other similar package managers the plain "root" name has been used. For example:<https://anaconda.org/conda-forge/root/>, <https://github.com/spack/spack/tree/develop/var/spack/repos/builtin/packages/root>.

CernRoot (or CERNRoot?) would also be a suitable alternative. 

If there is a request to change it to a more specific name, I can make this change.

Reply:
Conan package names are all lowercase, so it could be `cernroot`

I'm torn about between following upstream names vs. avoiding generic names

What do other reviewers think about it?

Reply:
Add the hyphen :smile: -> `cern-root`

Reply:
It's a valid concern, https://github.com/search?l=C%2B%2B&q=root&type=repositories says "1,714 repository results" so I thing the usual "author-name" should do the trick here

Reply:
OK looks like the consensus is `cern-root`. I will make this change.

Reply:
Implemented by 3f54990